docs/sphinx/reference-libobs-graphics-graphics.rst

```diff
@@ -997,6 +997,60 @@ Texture Functions
 
 ---------------------
 
+.. type:: enum gs_dmabuf_flags
+
+   DMA-BUF capabilities:
+
+   - GS_DMABUF_FLAG_NONE
+   - GS_DMABUF_FLAG_IMPLICIT_MODIFIERS_SUPPORTED  - Renderer supports implicit modifiers
+
+---------------------
+
+.. function:: bool *gs_query_dmabuf_capabilities(enum gs_dmabuf_flags *dmabuf_flags, uint32_t **drm_formats, size_t *n_formats)
+
+   **Linux only:** Queries the capabilities for dmabufs.
+
+   DMA-BUFs allow graphic cards to optimize storing or rendering by storing
+   the buffer content in custom layouts. These layouts for a given format are
+   tracked by the modifier of the DMA-BUF. Since these layouts are a hardware
+   feature, it's up to the used graphics card and the used graphics api which
+   modifiers are supported for a given format.
+
+   This function queries all supported formats, stores them
+   as an array. It also provides the number of formats and other capabilities
+   signaled by flags in the corresponding arguments.
+
+   The caller of this function is required to free the array.
+
+   :param dmabuf_flags:	Pointer to receive a capability bitmap
+   :param drm_formats:  Pointer to receive an array of DRM formats
+   :param n_formats:	Pointer to receive the number of formats
+   :rtype:              bool
+
+---------------------
+
+.. function:: bool *gs_query_dmabuf_modifiers(uint32_t drm_format, uint64_t **modifiers, size_t *n_modifiers)
```

Comment:
`gs_query_dmabuf_modifiers_for_format()` since this function operates on `drm_format`